org.ekstep.question-1.0/editor/question-ctrl.js

```diff
@@ -427,11 +432,18 @@ angular.module('org.ekstep.question', ['org.ekstep.metadataform'])
     if($scope.questionData.templateType != templateType){
       $scope.questionData.templateType = templateType;
       $scope.showPreview();
-      $scope.templateChanged = true;
-      $('.template-warning-Message').fadeIn(1000);
-      $('.template-warning-Message').delay(5000).fadeOut(5000);
+      $scope.showToaster($scope.messages.layoutChangeMsg);
     }
   }
+
+  $scope.showToaster = function(msg){
+    $scope.displayToaster = true;
```

Comment:
Remove this  $scope.displayToaster variable used to show/hide (ng-show). Because you are playing with fadeIn or fadeOut animation to show/hide using jQuery